pyproject.toml

```diff
@@ -19,7 +19,7 @@ license = {text = "MIT"}
 "Bug Tracker" = "https://github.com/ERGO-Code/HiGHS/issues"
 
 [build-system]
-requires = ["meson-python"]
+requires = ["meson-python<0.14.0", "meson>=1.2.0"]
```

Comment:
Minor nit, the `meson.build` file should also define `meson_version`:
```meson
project('highs', 'cpp',
  version : '1.6.0',
  meson_version: '>= 1.2.0',
...
```

Reply:
Done